bundles/org.openhab.binding.keba/src/main/resources/OH-INF/thing/kecontact.xml

```diff
@@ -9,7 +9,8 @@
 		<description>A KeContact EV Charging Station</description>
 
 		<channels>
-			<channel id="enabled" typeId="enabled"/>
+			<channel id="enabledsystem" typeId="enabled"/>
+			<channel id="enableduser" typeId="enabled"/>
```

Comment:
Please update the documentation to describe these 2 channels.
Please provide upgrade instructions (XML file) for an automatic update of the existing things for users suing managed things.
Add a description here for each channel to override the generic description of the channel type (enabled) and then regenerate default translations property file.

Reply:
> --> Done

Maybe you did it but you did not push your changes if you did it.

Reply:
Strange, maybe I just forgot or it was interrupted and I did notice... sorry. :astonished: